Source/WebCore/html/parser/create-html-entity-table

```diff
@@ -109,60 +101,124 @@ namespace WebCore {
 namespace {
 """)
 
+assert len(entries) > 0, "Code assumes a non-empty entity array."
+def check_ascii(entity_string):
+    for c in entity_string:
+        code = ord(c)
+        assert 0 <= code <= 127, (c + " is not ASCII. Need to change type of storage from LChar to UChar to support this entity.")
+
+output_file.write("static const LChar staticEntityStringStorage[] = {\n")
+output_file.write("'")
+all_data = ""
+entity_offset = 0
+first_output = True
+saved_by_reusing = 0
 for entry in entries:
-    output_file.write("static const LChar %s[] = \"%s\";\n" % (convert_entity_to_cpp_name(entry[ENTITY]), entry[ENTITY]))
+    check_ascii(entry[ENTITY])
+    # Reuse substrings from earlier entries. This saves 1-2000
+    # characters, but it's O(n^2) and not very smart. The optimal
+    # solution has to solve the "Shortest Common Superstring" problem
+    # and that is NP-Complete or worse.
```

Comment:
How long does this take to run? Seeing O(n^2) here makes me slightly worried about build speed.

Reply:
Hmm.. Let me do a quick local build with and without these changes and report back. :-)

Reply:
WebKit ToT has some build failure errors in WebProcessProxy.h & WebExtensionContext.h and it is 2 AM here, will run it tomorrow morning. :-(

Reply:
I tried right now with WebKit ToT ([261219@main](https://commits.webkit.org/261219@main)) with --release flag with and without patch but I think I am doing something wrong and not doing clean build because first with patch, it took 27 minutes (because I refreshed to get all patches till ToT) and then next without patches took less than a minutes. I think my methodology is flawed. Appreciate if you can guide to force clean build.

Reply:
We should merge this. That sounds like too much trouble to me.

Reply:
@darinadler - Sorry I was away from laptop. Thanks for adding 'merge-queue'.